query/search.js

```diff
@@ -1,7 +1,8 @@
 var peliasQuery = require('pelias-query'),
     defaults = require('./search_defaults'),
     textParser = require('./text_parser'),
-    check = require('check-types');
+    check = require('check-types'),
+    _ = require('lodash');
```

Comment:
Is lodash actually required? `.every` and `.some` are methods on `Array`, so I think it's actually fine.